mmv1/products/bigquery/Dataset.yaml

```diff
@@ -33,13 +33,17 @@ examples:
     vars:
       dataset_id: 'example_dataset'
       account_name: 'bqowner'
+    ignore_read_extra:
```

Comment:
We know which fields these are- let's add them directly, rather than needing to configure them as extra: https://github.com/GoogleCloudPlatform/magic-modules/blob/33a4f950ff8b1be6101f681bb4b96ec9c4cf2ff1/mmv1/templates/terraform/examples/base_configs/test_file.go.erb#L46

Reply:
Changed.